example/README.md

```diff
@@ -0,0 +1,70 @@
+In this folder, we can see four main packages in different sub-folders:
+- register_certificate
+- register_contract
+- execute_contract
+- validate_ledger
+
+They demonstrate how to use Scalar DL Go Client SDK to send basic operation requests to Scalar DL networks.
+
+To build all of them
```

Comment:
Should we have `how to build` and `how to install` doc in the top level README? (like the other SDKs?)

Reply:
You are right.
I should have done the root folder's README first otherwise this README in the example folder seems a bit weird.

Please check this commit https://github.com/scalar-labs/scalardl-go-client-sdk/pull/20/commits/74c9d369ce2f7bc384e758d13fc2cdf3584f53fd (the README)